BO4E/BO/Wechsel.cs

```diff
@@ -0,0 +1,62 @@
+using BO4E.COM;
```

Comment:
Müssten wir das dann auch für unseren Zählewechsel irgendwie benutzen?

Reply:
Ich glaube für den Zählerwechsel nicht unbedingt, aber für die Stammdatenänderung in der nur ein Gerät gewechselt wird wäre das ggf. spannend. Wobei der ja eine Referenzierung auf die Zählernummer hätte und damit könnte man dann auch wieder das Zähler-BO verwenden was wir für den 17009-Fall nicht verwenden können.